Release: Prepare "Translation string freeze" step

hebasto opened the pull request:
This PR follows our [Release Process](https://github.com/bitcoin/bitcoin/blob/53a996f122663e271efa52c45b173613b8ac635e/doc/release-process.md).

It is required for the translation string freeze, as the following PRs introduced new translatable strings:
 - #31296:https://github.com/bitcoin/bitcoin/blob/7b4a1350dfd6b3892a9c314eeff28b22ef0c1a73/src/wallet/wallet.h#L945

- #31453:https://github.com/bitcoin/bitcoin/blob/7b4a1350dfd6b3892a9c314eeff28b22ef0c1a73/src/init.cpp#L1878-L1879

- #32896:https://github.com/bitcoin/bitcoin/blob/22e689587a7ae02f82ad2464017731f3b23b5363/src/wallet/spend.cpp#L288-L292

**Notes for reviewers:**

1. To reproduce the diff, run:
```
cmake --preset dev-mode
cmake --build build_dev_mode --target translate
```

2. The structure of `bitconstrings.cpp` has been altered due to #33209.

3. The diff in `bitcoin_en.xlf` contains many unrelated metadata changes, so it may be easier to verify the changes in `bitcoin_en.ts`.

janb84 reviewed (commented):
ACK d32fcd3df22671b633cae605ed06c6d0a353e2bf

PR updates translation strings, for 30.0 freeze. 

Ran the commands against this PR; got 0 changes (reproduced the results) ✅
Ran the commands against Master; got the changes as provided in this PR ✅

hebasto committed:
qt: Update `src/qt/locale/bitcoin_en.xlf` translation source file

Steps to reproduce the diff:
```
cmake --preset dev-mode
cmake --build build_dev_mode --target translate
```

janb84 reviewed (commented):
re ACK 0df2c3c42e8a3ee3da4ca6c4b9b9c28060bb1fe3

changes since last ACK: 
- rebase (churn is inline with changes from #33209 )
